### PR TITLE
✨[Frontend] Denylist for (lite) registration emails

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

As requested by Mar and @calys, an email denylist is used in ``sim4life.science`` and ``tiplite``

Blacklist

## Related issue/s

related to https://github.com/ITISFoundation/osparc-issues/issues/1701

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/resource/osparc/blacklist.json
Comment (pcrespov) [inline services/static-webserver/client/source/resource/osparc/blacklist.json:1]: I would rename it as blocklist, bannedlist, denylist :-)

File: services/static-webserver/client/source/resource/osparc/blacklist.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
+{
+  "lite": [
```
Comment (pcrespov) [inline services/static-webserver/client/source/resource/osparc/blacklist.json:2]: why not apply these to *all* lite the products?

Comment (odeimaiz) [reply services/static-webserver/client/source/resource/osparc/blacklist.json:2]: It applies to all ``lite`` products that have a request account form. ``s4llite``, for now goes through the form in the ZMT website.

File: services/static-webserver/client/source/resource/osparc/blacklist.json
Comment (odeimaiz) [reply services/static-webserver/client/source/resource/osparc/blacklist.json:1]: Working group for the filename?

Comment (pcrespov) [reply services/static-webserver/client/source/resource/osparc/blacklist.json:1]: i think being politically correct in this case does not cost that much :-)

Comment (odeimaiz) [reply services/static-webserver/client/source/resource/osparc/blacklist.json:1]: oh, I didn't even realize 😱

Comment (pcrespov) [reply services/static-webserver/client/source/resource/osparc/blacklist.json:1]: https://dev.to/vinibrsl/github-to-replace-racially-loaded-terms-master-slave-blacklist-and-whitelist-mnp

**Review by jsaq007 — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/auth/core/Utils.js [hunk 2]
```diff
@@ -51,6 +51,23 @@ qx.Class.define("osparc.auth.core.Utils", {
       return isValid;
     },
 
+    denylistEmailValidator: function(denylist) {
```
Comment (GitHK) [inline services/static-webserver/client/source/class/osparc/auth/core/Utils.js:54]: Q: should this check not be done by the backend? I could disable this validation and bypass it.

Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/auth/core/Utils.js:54]: Ideally, yes.

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**